models/ht_connection.h

```diff
@@ -178,15 +180,12 @@ HTConnection< targetidentifierT >::send( Event& e,
   double t_lastspike,
   const CommonSynapseProperties& )
 {
-  double h = e.get_stamp().get_ms() - t_lastspike;
-  Node* target = get_target( t );
-  // t_lastspike_ = 0 initially
-
   // propagation t_lastspike -> t_spike, t_lastspike_ = 0 initially, p_ = 1
```

Comment:
No, here `t_spike` is actually the time of the spike passing through the synapse.